purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -2520,14 +2523,19 @@ class PurchasesTest {
             offeringIdentifier = null,
             type = PurchaseType.INAPP
         )
-        capturedPurchasesUpdatedListener.captured.onPurchasesUpdated(getMockedPurchaseList(
-            sku,
-            purchaseToken,
-            PurchaseType.INAPP,
-            null
-        ))
-        capturedConsumeResponseListener.captured.invoke(BillingClient.BillingResponseCode.SERVICE_UNAVAILABLE.buildResult(), purchaseToken)
-        verify (exactly = 0) {
+        capturedPurchasesUpdatedListener.captured.onPurchasesUpdated(
```

Comment:
what's the character limit on the formatter? Is it 80?

Reply:
I usually go with 120 because most monitors will still show 120 characters in one line and it's the limit for github's diff at default zoom

Reply:
Yes, it's 120. There's a function in Android studio that will put arguments in multiple lines and I tend to overuse it.